packages/webpack-extraction-plugin/src/index.ts

```diff
@@ -1 +1,2 @@
 export { GriffelCSSExtractionPlugin } from './GriffelCSSExtractionPlugin';
+export { createGriffelNextExtractPlugin as nextPlugin } from './nextPlugin';
```

Comment:
```suggestion
export { withGriffelCSSExtraction } from './withGriffelCSSExtraction';
```

As plugins in Next.js follow `withX` pattern, let's so follow it.